docs/components/pages/landing-page.md

```diff
@@ -0,0 +1,183 @@
+# Landing Page
+
+This component provides an application landing page, which is usually the first
+page shown to a user.
+
+## Usage ---
+
+The Landing Page is a part of an overarching concept for guiding a user into a
+product. Legal guidelines are crucial for the sign-up and sign-in process.
+
+### When to use
+
+- A user has to sign-up or sign-in before using the product.
+- A user has to be informed about product usage conditions.
+
+## Design ---
+
+The Landing Page provides the first impression of a product to the users. It has
+to meet multiple goals:
+
+- **Functionality:** This is the major driver for having a landing page. The
+  landing page should present the functionality in an easy way.
+- **Legal information** Necessary legal artifacts must be provided to the user.
+- **Market Communication** (MarCom): The landing page must provide a (visual)
+  bridge to the marketing material of a product.
+- **DevOps information**  The landing page shall inform about any product
+  usage interruptive events as planned maintenance activities.
+
+Covering those need solely for large screen might lead to overloaded content.
+The design is therefore done using the **mobile first** approach.
+
+![Landing Page](images/landing-page.png)
+
+> 1. Brand logo, 2. Language selector, 3. Marketing image, 4. Product info, 5.
+> Marketing claim (Optional), 6. Functional UI elements, 7. Registration
+> (Optional), 8. Additional legal info, 9. Footer with version (optional), copyright and legal links
+
+### Functional UI Elements
```

Comment:
Seems like an empty headline. I think we might also wanna write Elements lower case here.